src/rtk/features/apiDelegation/overviewOrg/overviewOrgSlice.ts

```diff
@@ -334,12 +343,13 @@ const overviewOrgSlice = createSlice({
         state.loading = false;
       })
       .addCase(fetchOverviewOrgsReceived.rejected, (state, action) => {
+        const payload = action.payload as RejectedPayload;
         state.loading = true;
-        state.error.statusCode = String(action.payload?.response?.status) ?? 'Unknown code';
```

Comment:
The `??` makes no sense here; `String(action.payload?.response?.status)` will be either status as a string, 'undefined' or ''